docs/intro/advanced-tutorial.rst

```diff
@@ -62,7 +62,7 @@ Suppose we have the following Page Object:
 
             # Simulates clicking on a button that says "View All Images"
             response: web_poet.HttpResponse = await self.http.get(
-                f"https://api.example.com/v2/images?id={item['product_id']}"
+                f"https://api.toscrape.com/v2/images?id={item['product_id']}"
```

Comment:
This is not a working URL; is it possible to have a real example? I think the idea to use toscrape.com domain is to actually make the examples runnable.

Reply:
I agree @kmike  . Though, the situation here is that we do not have a toscrape.com website that supports API calls. Now there are websites that support dummy/test API calls but the mandate of the issue was that we should not be using other external domains.

Can we setup an API based toscrape website?

Reply:
Not sure how it fits the example, but I think http://quotes.toscrape.com/scroll example uses an AJAX call.